### ISSUE TITLE
test

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/internship#333]
Draft of Template Issue: Research: Interviews: Make Issues
---
name: TWE: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [replace with -TYPE OF INTERVIEW (E.G., INTAKE OR EXIT)] Interviews: Make Issues and Track Completion
about: The parent issue that tracks issues for all interviews of a specific research plan
title: TWE: Research: Interviews: Make Issues
labels: 'feature: board / google drive / tools', 'feature: research', 'feature: missing', 'size: 1pt', 'role: UX research lead', 'milestone: missing' 'epic' 
assignees: 
---

### Dependencies
- [ ] An interview training guide has been created and approved [replace with - ISSUE LINK AND ADD LINK TO GUIDE IN RESOURCE # 5]

### Overview
We need to create an epic issue that tracks each [Replace TYPE OF PARTICIPANT] [replace with - TYPE OF INTERVIEW (E.G., INTAKE OR EXIT)] interview to make sure we are getting all the steps uniform.

#### Customize the Issues
by adding the Research Plan No, Type of Participant, and Type of Interview (including title). Replace the keyword strings (including text and the brackets) below with the appropriate values by using the browser search (Mac: Command+F  PC: Ctrl+F)
- Research Plan Number
   - [ ] Search for `[Replace 000]`
   - [ ] Replace with the number of your research plan (e.g., 006)
- Type of Participant
   - [ ] Search for `[Replace TYPE OF PARTICIPANT]`
   - [ ] Replace with the name of the participant type (e.g., Intern)
 - Type of Interview
   - [ ] Search for `[Replace TYPE OF INTERVIEW]`
   - [ ] Replace with the name of the interview type (e.g., Intake or Exit)

### Customize the issue settings (located right side bar)
change the issue settings to ensure the issue is properly categorized and easy to manage.
- [ ] Under Assignees, assign yourself.
- [ ] Under Labels, replace the `Research: 000: missing` with the label of associated with this research plan (e.g., Research: RP001) (helps you find related issues in the same workflow) and remove the corresponding missing label.
- [ ] Under Projects, click gear, choose "Repository", choose Program Management.

#### Customize the Resources section of this issue (below)
- No. 1 no change needed
- Customize No. 2 [RP[Replace 000] Wiki Page
   - [ ] Go to the wiki page: Research Output Overview (# 1 link in resources) 
   - [ ] Find the relevant wiki page for RP[Replace 000]. 
   - [ ] Copy the link for the wiki page.
   - [ ] Update the [RP[Replace 000] Wiki Page] with the link you just copied into parentheses at the end of the line.
-  No. 3 - no change needed
-  Customize No. 4 link in resources: [Google Drive RP[Replace 000] Folder]
   - [ ] Open the Google Drive's Research by Type Folder (# 3 link in Resources)
   - [ ] Open the [Replace TYPE OF PARTICIPANT] Folder (e.g., 1. Mentor, 2. Intern, etc.) 
   - [ ] Right-click on the [Replace 00] folder and copy the link. (e.g., RP6 - Intern Intake Interviews)
   - [ ] Update [Google Drive RP[Replace 000] Folder] add the link you just copied into parentheses at the end of the line.
-  Customize No. 5 link in resources: TWE: RP[Replace 000] Interview Instructions
   - [ ] Open the Google Drive's Research by Type Folder (# 3 link in Resources)
   - [ ] Open the [Replace TYPE OF PARTICIPANT] Folder (e.g., 1. Mentor, 2. Intern, etc.) 
   - [ ] Open the [Replace 00] folder
   - [ ] copy the link. (e.g., RP6 - Intern Intake Interviews)
   - [ ] Right-click on the Interview Instructions document and copy the link
   - [ ] Update the TWE: RP[Replace 000] Interview Instructions with the link you just copied into parentheses at the end of the line.
- No 6 -  no change needed
- No 7 - link the roll call spreadsheet
- No 8 no change needed
- Customize No. 9 link in resources: Form
   - [ ] Get link to form from related issue.  If no issue created, use the [Replace NAME OF TEMPLATE] to create the form
   - [ ] Save that link to the resources of this issue 1. [TWE: RP[Replace 000]: Interview Recording Upload Form Template] (Resource # 10)

#### Create an issue for each interview.
- [ ] IS[replace with YY]: RP[Replace 000]: [replace with -TYPE OF INTERVIEW] Interviews: [Replace TYPE OF PARTICIPANT] # 1
- [ ] IS[replace with YY]: RP[Replace 000]: [replace with -TYPE OF INTERVIEW] Interviews: [Replace TYPE OF PARTICIPANT] # 2
- [ ] Replace this action item with as many interview issues that you need. 
   - [ ] Hover over the action item to the left so that the target circle appears (the action item should be the exact name of the issue, e.g., `IS22: RP006: Exit Interviews: Intern # 1`
    - [ ] Click "Convert to Issue"
    - [ ] Copy and paste the gray template issue text into the issue that you just created.
    - [ ] Click "Submit New Issue"
- [ ] Add the issue numbers/links to the interview spreadsheet (Resource # 6)

#### Schedule the interviews 
(Note if you have to create more than one interview, you may have to do the following steps several times)
- [ ] Go to the UI/UX wiki page for setting up interviews on Hack for LA Zoom accounts (Resource Link 10).
- [ ] Copy the action items to this issue in a comment and check off each item.
- [ ] Update the related interview issue with the time and Zoom link information.

#### Follow-up steps
     - [ ] Check whether all recordings have been uploaded to the correct folder and that any personally identifiable information is NOT stored in the shared drive.
     - [ ] Contact the UX researchers who uploaded the recordings and ask them to delete their files. 

### Create documents
- TWE: Interview Session and Roll Call Table Template
  - [ ] Click on Resource No 6: TWE: Template: Interview Session Roll Call Spreadsheet
  - [ ] Click on the `Use Template` button to save it to your own drive
  - [ ] Move the file into the Internship's shared drive in this Research Plan's Google Drive folder (Resource No 4).
  - [ ] Copy link to the template issue text in gray (Resource # 11, Item B)
-  Create Easy Reto
    - [ ] In Resources, click on 1password link (# 7) to find the login information within the Internship-UX vault for Easy Retro
    - [ ] Log into EasyRetro using the login information.
    - [ ] Create an Easy Retro for Q&A that occurs during the interview. 

#### Customize templates
-  Copy link to TWE: RP[Replace 000] Interview Instructions
   - [ ] Open the Google Drive's Research by Type Folder (# 3 link in Resources)
   - [ ] Open the [Replace TYPE OF PARTICIPANT] Folder (e.g., 1. Mentor, 2. Intern, etc.) 
   - [ ] Copy the link to the Interview Instructions
   - [ ] Update Resource # 11, Item A 
- Copy link for TWE: Interview Session and Roll Call Table  to 
   - [ ] Open the Google Drive's Research by Type Folder (# 3 link in Resources)
   - [ ] Open the [Replace TYPE OF PARTICIPANT] Folder (e.g., 1. Mentor, 2. Intern, etc.) 
   - [ ] Open the [Replace 00] folder
   - [ ] Copy the link to the Session and Roll Call Table Spreadsheet
   - [ ] Update (Resource # 11, Item B)
 - Copy link for easy retro to (Resource # 11, Item C) 
 - Copy link to Upload Form to the template issue text in gray (Resource # 11, Item D)
 - Copy link to roll call spreadsheet (link for Resource # 11, Item B)

### Resources
1. [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1. [RP[Replace 000] Wiki Page] 
1. [Google Drive Research by Type Folder](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=sharing)
1. [Google Drive RP[Replace 000] Folder]
1. [TWE: RP[Replace 000] Interview Instructions](https://docs.google.com/document/d/1YKJZHejAAvX19NfPh1AKC4k3CuX0nrJQq9N3BXBAB2U/template/preview) 
1. [TWE: Template: Interview Session Roll Call Spreadsheet](https://docs.google.com/spreadsheets/d/1dmJJLaBDBiZWy4-dmdxG3XqP4OemR5aJTOScD_GsoLc/template/preview) 
1.  [TWE: RP[Replace 000]: Interview Session Roll Call Spreadsheet]
1. [1password.com](https://1password.com/)
1. [Easy Retro Link] 
1. [Interview Recording Upload Form Template](https://docs.google.com/forms/d/1FsDgBvpkjfJzmkxFl80I9HWenoOwpvst7oMAaDFgPEg/edit) 
1. [TWE: RP[Replace 000]: Interview Recording Upload Form Template]
1. [UI/UX Wiki: Setup Interviews on Hack for LA Zoom Accounts](https://github.com/hackforla/UI-UX/wiki/Setup-Interviews-on-Hack-for-LA-Zoom-accounts)

#### Resource # 11 Template for IS[YY] [TYPE & TIMING] Interview Issues
```
### Overview
We need to track the interview for [Intern's Number] so that we don't drop the ball or lose data about it.

### Action Items
- [ ] If you are conducting the interview on your own and not in a breakout room
     - [ ] 5 min prior, login to 1password and get hostkey for Zoom [Zoom meeting room#]
     - [ ] Join the meeting via the calendar link in your invite or here in this issue
     - [ ] Claim host
- [ ] When they enter room, begin consent process, and ask if we can record and transcribe.
- [ ] Turn on transcript.
- [ ] Use script in interview.
- [ ] Upload recording and transcript and a chat file if applicable using the Interview Recording Upload Form
- [ ] Verify with UXR lead that recordings have been uploaded correctly
- [ ] Delete recordings (only after UXR lead directs you to do so).

### Resources/Instructions
#### Interview Links and Instructions
- A) [RP[Replace 000]: Interviews Instructions]
- B) [TWE: RP[Replace 000]: [Replace TYPE OF PARTICIPANT][Replace INTERVIEW TYPE] Interview Session and Roll Call Table]
- C) [Easy Retro link for Q&A that occurs during interview]
- D) [[Replace TYPE OF PARTICIPANT][Replace INTERVIEW TYPE] Interview Recording Upload Form]

#### Interview details
- Zoom link for interview
- Date and time for interview:

#### Other links
- [Code for America's Guide on Qualitative Research](https://info.codeforamerica.org/qualitative-research)

#### Text once you are ready to send invite update
We have added the zoom invite info and the panelist for your interview.  Looking forward to seeing you! 
```
